packages/core/src/error-reporter.ts

```diff
@@ -1,25 +1,22 @@
-import { GlobalConfig } from '@n8n/config';
 import type { NodeOptions } from '@sentry/node';
 import type { ErrorEvent, EventHint } from '@sentry/types';
 import { AxiosError } from 'axios';
 import { ApplicationError, LoggerProxy, type ReportingOptions } from 'n8n-workflow';
 import { createHash } from 'node:crypto';
 import { Service } from 'typedi';
 
-import { InstanceSettings } from './InstanceSettings';
+import type { InstanceType } from './InstanceSettings';
 
 @Service()
```

Comment:
Since this is not provided by DI container anymore
```suggestion
@Service()
```